cmd/peek-submission/main.go

```diff
@@ -61,43 +44,10 @@ func main() {
 		TargetSubmission: args.TargetSubmission,
 	}
 
-	requestMap := map[string]any{
-		server.ENDPOINT_KEY: core.NewEndpoint(`courses/assignments/submissions/fetch/user/peek`),
-		server.REQUEST_KEY:  request,
-	}
-
-	jsonRequest := util.MustToJSONIndent(requestMap)
-	jsonBytes := []byte(jsonRequest)
-	err = util.WriteToNetworkConnection(connection, jsonBytes)
+	response, err := cmd.SendCMDRequest(`courses/assignments/submissions/fetch/user/peek`, request)
 	if err != nil {
-		log.Fatal("Failed to write the request to the unix socket.", err)
+		log.Fatal("Failed to send the CMD request.", err)
 	}
 
-	responseBuffer, err := util.ReadFromNetworkConnection(connection)
-	if err != nil {
-		log.Fatal("Failed to read the response from the unix socket.", err)
-	}
-
-	var response core.APIResponse
-	util.MustJSONFromBytes(responseBuffer, &response)
-
-	if !response.Success {
-		output := response.Message
-		if !args.Short {
-			output = util.MustToJSONIndent(response)
-		}
-
-		fmt.Println(output)
-
-		util.Exit(2)
-		return
-	}
-
-	if args.Short {
-		var responseContent submissions.FetchUserPeekResponse
-		util.MustJSONFromString(util.MustToJSON(response.Content), &responseContent)
-		fmt.Println(util.MustToJSONIndent(responseContent))
-	} else {
-		fmt.Println(util.MustToJSONIndent(response))
-	}
+	cmd.PrintCMDResponse(response, submissions.FetchUserPeekResponse{}, args.Short)
```

Comment:
What about wrapping together `cmd.SendCMDRequest()` and `cmd.PrintCMDResponse()` into one call?